test/common/quic/active_quic_listener_test.cc

```diff
@@ -352,7 +360,29 @@ TEST_P(ActiveQuicListenerTest, ReceiveCHLO) {
   readFromClientSockets();
 }
 
+TEST_P(ActiveQuicListenerTest, ConfigureReasonableInitialFlowControlWindow) {
+  // These initial flow control windows should be accepted by both Google QUIC and IETF QUIC.
+  connection_window_size_ = 1024 * 64;
+  stream_window_size_ = 1024 * 32;
```

Comment:
nit: the more relevant component of these 2 constants is the 64 or 32, not the 1024.  I recommend flipping the order so they are easier to read and are more consistent with similar computations elsewhere in the code:
64 * 1024
32 * 1024